### ISSUE TITLE
Handling missing values in tabular data when using image + tabular inputs

### ISSUE THREAD

**Issue by xuechunlianglab** [jrzaurin/pytorch-widedeep#251]
Hello, thank you very much for your outstanding work on this project!

I would like to apply your framework to predict cancer occurrence using paired image + structured tabular data. However, some of the tabular features may contain missing values (for example, some samples do not have age information).

My questions are:

Does the framework currently support handling missing values in tabular data?

If not, do you recommend any strategies (e.g., imputation, masking, special embeddings) for integrating such incomplete tabular features with the image encoder?

Thank you again for your great contribution, and I look forward to your advice.

**Comment by jrzaurin** [comment]
Hey @xuechunlianglab 

the library does not support handling missing values per se, this is something you would have to handle yourself at a pre-processing stage. 

As recommending strategies, it really depends on the meaning of having a NaN. 

for example, if having NaN is simply lack of data collection, then some form of clever imputation might be worth exploring (like the `IterativeImputer` in sklearn.

However, if the NaN has meaning, such as for example absence of data for that particular column given all others, then I tend to impute it as a different category if is a categorical column or as a different number is is a continuous col, with the aim to teach the model that the value there is different and has meaning
